sample_consensus/include/pcl/sample_consensus/sac_model.h

```diff
@@ -555,14 +522,11 @@ namespace pcl
       /** Data containing a shuffled version of the indices. This is used and modified when drawing samples. */
       std::vector<int> shuffled_indices_;
 
-      /** \brief Boost-based random number generator algorithm. */
-      boost::mt19937 rng_alg_;
-
-      /** \brief Boost-based random number generator distribution. */
-      boost::shared_ptr<boost::uniform_int<> > rng_dist_;
+      /** \brief Random number generator algorithm. */
+      std::mt19937 rng_;
 
-      /** \brief Boost-based random number generator. */
-      boost::shared_ptr<boost::variate_generator< boost::mt19937&, boost::uniform_int<> > > rng_gen_;
+      /** \brief Random number generator distribution. */
+      std::shared_ptr<std::uniform_int_distribution<> > rng_dist_;
```

Comment:
Maybe I overlooked something, but I don't see any reason for this to be a shared pointer.